frontend/packages/client/package.json

```diff
@@ -6,6 +6,8 @@
     "@creativebulma/bulma-tooltip": "^1.2.0",
     "@onflow/fcl": "^1.2.0",
     "@onflow/six-transfer-tokens": "0.0.8",
+    "@tanstack/react-query": "^4.1.0",
+    "@tanstack/react-query-devtools": "4.0.11-beta.0",
```

Comment:
Had to install beta version bc 4.0.11 was not working.  Read this here: https://github.com/TanStack/query/issues/3916#issuecomment-1213172411
I think it's not an issue because we are only using this for local dev, and dev-tools are not included in prod bundle